src/components/calcite-tile-select/calcite-tile-select.tsx

```diff
@@ -193,13 +196,17 @@ export class CalciteTileSelect {
     this.input.checked = this.checked;
     this.input.disabled = this.disabled;
     this.input.hidden = this.hidden;
-    this.input.id = this.el.id;
```

Comment:
@eriklharper do you know why this was being set to the element's id? All ids should be unique so I'm not sure why this was happening.

Reply:
Definitely was a mistake to set that on this component originally.  IIRC it was to allow native label support so that a wrapping `<label>` would properly focus the input.